server/game/gamesteps/conflict/initiateconflictprompt.js

```diff
@@ -178,7 +177,21 @@ class InitiateConflictPrompt extends UiPrompt {
     }
 
     recalculateCovert() {
+        // remove any selected defenders with covert
+        this.selectedDefenders.map(card => {
```

Comment:
I don't really like using `.map` to iterate over a function....

Reply:
This is my ignorance of JS array functions.

What is best to use `_.each`?

Reply:
You could use `_.each` or a for loop